### PR TITLE
tests: run ToT and stable Chrome for smoke tests in github workflow

### PR DESCRIPTION
**Pull request by connorjclark**
Currently only appveyor runs ToT Chrome. Better to do both ToT and stable in GitHub CI.

#10988 will be necessary to avoid ~30m long builds.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -37,6 +37,13 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install protobuf==3.7.1
+    
+    # Chrome Stable is already installed by default.
+    - name: Install Chrome ToT
+      run: bash lighthouse-core/scripts/download-chrome.sh
```
Comment (connorjclark) [inline .github/workflows/ci.yml:43]: AFAIK there's no caching w/ this approach. I don't think we want to cache ToT installation, right?

Comment (patrickhulce) [reply .github/workflows/ci.yml:43]: I could see us caching within the same day or something, but I agree we don't want to cache for very long. either way followup seems fine.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -37,6 +37,13 @@ jobs:
...
+      env:
+        CHROME_PATH: /home/runner/chrome-tot
```
Comment (patrickhulce) [inline .github/workflows/ci.yml:45]:
```suggestion
        CHROME_PATH: /home/runner/chrome-linux-tot
```

even if we're not caching, feels like they should match

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -37,6 +37,13 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install protobuf==3.7.1
+    
+    # Chrome Stable is already installed by default.
+    - name: Install Chrome ToT
+      run: bash lighthouse-core/scripts/download-chrome.sh
+      env:
```
Comment (patrickhulce) [inline .github/workflows/ci.yml:44]:
```suggestion
      working-directory: /home/runner
      env:
```

optional, feels weird to download this in repo root, might as well just download it where we want it

**Comment by connorjclark** [comment]
OK let's invalidate all these other PRs now :)

**Pull request merged**